lib/max_subarray.py

```diff
@@ -2,11 +2,32 @@
 def max_sub_array(nums):
     """ Returns the max subarray of the given list of numbers.
         Returns 0 if  nums is None or an empty list.
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(1)
+        Space Complexity: O(1)
```

Comment:
✨ Your implementation works, however time complexity would be O(n) where n is the length of `nums` as you are looping through the length of `nums`